.github/build_openassetio_mediacreation/action.yml

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 The Foundry Visionmongers Ltd
+
+# Composite action for reuse within other workflows.
+# Builds OpenAssetIO.
+# Should be run on a ghcr.io/openassetio/openassetio-build container.
+
+name: Build OpenAssetio-MediaCreation
+description: Builds OpenAssetIO-MediaCreation and publishes an artifact
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout OpenAssetIO-Mediacreation
+      uses: actions/checkout@v3
+      with:
+        repository: elliotcmorris/OpenAssetIO-MediaCreation
+        path: openassetio-mediacreation-checkout
+        ref: work/cmake-project #Change before commit
+
+    - name: Install Traitgen
+      shell: bash
+      run: |
+        python -m pip install git+https://github.com/feltech/OpenAssetIO-TraitGen@work/11-cppCodeGen
```

Comment:
[Fix] Remember to update

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-Test-CMake/pull/7/commits/7b8c3a796139a45a7ebc77352c9b2bbf13ee27ed